tests/test_gauss_pdf_and_cdf.py

```diff
@@ -0,0 +1,30 @@
+from os.path import exists
+from src.gauss_pdf_and_cdf import (GaussianProbabilisticSetting,
+                                   BetaProbabilisticSetting)
+import cv2
+
+
+def show_or_save_figure(distribution_type):
```

Comment:
- Add type annotation and docstring
- Actually, this function tests all functions related to figures. You should rename it to clear one.